### DISCUSSION TITLE
How to obtain the content of the current line, including terminal characters

### DISCUSSION THREAD

**Discussion by TheBlindM** [sourcelair/xterm.js#4793] [Q&A] [answered]
## Details
- Browser and browser version: 
- OS version: win10
- xterm.js version: 
5.1

### Steps to reproduce
```
 const currentLine = term.buffer.active.getLine(term.buffer.active.cursorY);
const currentInput = currentLine.translateToString(true);
```
output:
```
 (base) beavers@beavers-home:~$ 

```
expect：
```
(base) ]0;beavers@beavers-home: ~beavers@beavers-home:~$ 
```

**Comment by Tyriar** [comment, marked as answer]
In order to get sequences, you want to either record all data going into the terminal, or use the [serialize addon](https://github.com/xtermjs/xterm.js/blob/master/addons/xterm-addon-serialize/README.md)
